apps/css-workshop/src/pages/checkbox.astro

```diff
@@ -78,435 +55,228 @@ import Layout from './_layout.astro';
+          </div>
+          <div>Warning message.</div>
         </div>
-        <div>Warning message.</div>
       </div>
-    </div>
 
-    <div class='iui-input-grid iui-input-group-wrapper'>
-      <div class='iui-input-label'>Checkbox group label</div>
-      <div class='iui-input-group'>
-        <label class='iui-checkbox-wrapper'>
-          <input class='iui-checkbox' type='checkbox' checked />
-          <span class='iui-checkbox-label'>Option 1</span>
-        </label>
-        <label class='iui-checkbox-wrapper'>
-          <input class='iui-checkbox' type='checkbox' />
-          <span class='iui-checkbox-label'>Option 2</span>
-        </label>
-        <label class='iui-checkbox-wrapper iui-disabled'>
-          <input class='iui-checkbox' type='checkbox' checked disabled />
-          <span class='iui-checkbox-label'>Option 3</span>
-        </label>
-        <label class='iui-checkbox-wrapper iui-disabled'>
-          <input class='iui-checkbox' type='checkbox' disabled />
-          <span class='iui-checkbox-label'>Option 4</span>
-        </label>
-      </div>
-      <div class='iui-status-message' data-iui-status='negative'>
-        <div class='iui-svg-icon'>
-          <svg-status-error class='iui-svg-icon' aria-hidden='true'></svg-status-error>
+      <div class='iui-input-grid iui-input-group-wrapper'>
+        <div class='iui-input-label'>Checkbox group label</div>
+        <div class='iui-input-group'>
+          <Checkbox_ label='Option 1' checked />
+          <Checkbox_ label='Option 2' />
+          <Checkbox_ label='Option 3' disabled checked />
+          <Checkbox_ label='Option 4' disabled />
+        </div>
+        <div class='iui-status-message' data-iui-status='negative'>
+          <div class='iui-svg-icon'>
+            <svg-status-error class='iui-svg-icon' aria-hidden='true'></svg-status-error>
+          </div>
+          <div>Negative message.</div>
         </div>
-        <div>Negative message.</div>
       </div>
-    </div>
 
-    <div class='iui-input-grid iui-input-group-wrapper'>
-      <div class='iui-input-label'>Checkbox group label</div>
-      <div class='iui-input-group'>
-        <label class='iui-checkbox-wrapper'>
-          <input class='iui-checkbox' type='checkbox' checked />
-          <span class='iui-checkbox-label'>
-            <svg-smiley-happy-very aria-hidden='true' style='display: flex;'
-            ></svg-smiley-happy-very>
-          </span>
-        </label>
-        <label class='iui-checkbox-wrapper'>
-          <input class='iui-checkbox' type='checkbox' />
-          <span class='iui-checkbox-label'>
+      <div class='iui-input-grid iui-input-group-wrapper'>
+        <div class='iui-input-label'>Checkbox group label</div>
+        <div class='iui-input-group'>
+          <Checkbox_ checked>
             <svg-smiley-happy aria-hidden='true' style='display: flex;'></svg-smiley-happy>
```

Comment:
I think this should be `svg-smiley-happy-very`, as it was before. Not sure why the test didn't fail. Maybe the change amount was below the threshold?

Reply:
It didn't have the very happy variant before. I can add it tho

Reply:
Yeah I can add that back